src/hal-backend-ml-vivante.cc

```diff
@@ -369,6 +369,9 @@ _json_release_neural_network (vivante_handle_s *self)
 {
   if (self->graph) {
     vsi_nn_ReleaseGraph (&self->graph);
+    if (self->graph->ctx) {
```

Comment:
This may reference NULL ptr (self->graph)

LINE 673 of
https://github.com/zjd1988/TIM-VX-python/blob/af0d0c02bafbd9ddfadf7ff5c008787c0ba4686f/src/tim/vx/internal/src/vsi_nn_graph.c#L624